roles/confluent.variables/vars/main.yml

```diff
@@ -422,14 +418,22 @@ kafka_connect_properties:
                             true, sasl_plain_users.kafka_connect.principal, sasl_plain_users.kafka_connect.password, sasl_scram_users.kafka_connect.principal, sasl_scram_users.kafka_connect.password,
                             kerberos_kafka_broker_primary|default('kafka'), kafka_connect_keytab_path, kafka_connect_kerberos_principal|default('kafka'),
                             kafka_connect_ldap_user, kafka_connect_ldap_password, mds_bootstrap_server_urls) }}"
-  producer_monitoring_interceptor:
+  monitoring_interceptor:
+    # TODO do we want a flag for this feature
```

Comment:
add monitoring_interceptor_enabled and <component>_monitoring_interceptor_enabled vars